homeassistant/components/vallox/sensor.py

```diff
@@ -56,7 +56,7 @@ def __init__(
         self._attr_unique_id = f"{uuid}-{description.key}"
 
     @property
-    def native_value(self) -> StateType:
+    def native_value(self) -> StateType | datetime:
```

Comment:
I realize this is a larger change but should `StateType` just include `datetime` now that we want datetimes instead of strings?

Reply:
`StateType` is part of the generic entity class while `datetime` is part of the sensor class.

Reply:
`SensorStateType` would be an option.

Reply:
Thanks for the explanation, i'm good with this as is given that context.